### DISCUSSION TITLE
Anyone got an alignment file?

### DISCUSSION THREAD

**Discussion by aspencoyle** [sr320/course-fish546-2021#108] [Q&A] [answered]
Hi! I'm looking to visualize my data in IGV, and...I'm not sure if I can? I don't have any SAM or BAM files, and despite some pretty thorough searching, have failed to find any C. bairdi or Hematodinium SAM or BAM files used as part of previous steps!

I'm also totally up for visualizing the data using some random data files, just as practice if none are available!

**Comment by sr320** [comment, marked as answer]
Run a quick bowtie of one of your libraries to the genome....
https://owl.fish.washington.edu/halfshell/genomic-databank/cbai_genome_v1.01.fasta

**Comment by sr320** [comment]
OR... 
you could run this script to derive a GFF to visualize https://github.com/RobertsLab/code/blob/master/script-box/2_Blast2Gff.pl
